ui/desktop/app/routes/scopes.js

```diff
@@ -23,9 +42,59 @@ export default class ScopesRoute extends Route {
    * @param {string} params.scope_id
    * @return {Promise{[ScopeModel]}}
    */
-  model() {
+  async model() {
     // NOTE:  In the absence of a `scope_id` query parameter, this endpoint is
     // expected to default to the global scope, thus returning org scopes.
     return this.store.query('scope', {}).catch(() => A([]));
   }
+
+  /**
+   * Adds `isPaginationSupported`, `downloadLink`, and
+   * `downloadError` to the controller.
+   * @param {Controller} controller
+   */
+  async setupController(controller) {
+    super.setupController(...arguments);
+
+    controller.set('isPaginationSupported', this.isPaginationSupported);
+
+    let downloadLink;
+    let downloadError = false;
+
+    if (!this.isPaginationSupported) {
+      const metaDataUrl =
+        'https://api.releases.hashicorp.com/v1/releases/boundary-desktop/1.7.1';
```

Comment:
None blocking comment: thinking loud here, take with a pinch of salt.

The version specified in `metaDataUrl` which is `1.7.1` has to be always the latest version of our DC client or as long as is `1.7.1` is fine? asking to know if this is something we should add to our release guide. I.e: Update the version of suggested DC for unsupported pagination purposes.
Thoughts? 

Reply:
This url will not get updated again. We are using the 1.7.1 release of the boundary desktop because that will be the last desktop app that will support older controllers that do not support pagination. We will likely need to update how the autoUpdater to stop let the user know they need to update if the controller is an older controller as well.

Reply:
@cameronperera Thank you for the clarification. That makes sense :smiley: